### DISCUSSION TITLE
Has anyone deployed this on Azure WebApp

### DISCUSSION THREAD

**Discussion by GrooveDevops** [postalserver/postal#1700] [Help with using Postal]
Hi there,

I am trying to deploy this postal setup on azure webapp, I believe it needs to be done via docker but don't know how this should be done...... if anyone here who have done the same please ping on this thread 

Thanks

**Comment by willpower232** [comment]
I don't think you'll be able to make use of port 25 which is pretty crucial so I don't think this will be possible.

**Reply by GrooveDevops** [reply]
For that, we can open port.... but question is how to deploy

**Comment by elibroftw** [comment]
Yep would love to know how to deploy on Azure as we have Azure credits but are spending $15/month on email for postmark.
